weave

```diff
@@ -1717,13 +1773,26 @@ launch_router() {
             --no-restart)
                 RESTART_POLICY=
                 ;;
+            --awsvpc)
+                WEAVE_NO_FASTDP=1
+                AWSVPC_ARGS="--awsvpc"
+                AWSVPC=1
+                ;;
             *)
                 ARGS="$ARGS '$(echo "$1" | sed "s|'|'\"'\"'|g")'"
                 ;;
         esac
         shift
     done
     eval "set -- $ARGS"
+
+    create_bridge
+    docker_bridge_ip
```

Comment:
Re redundant `docker_bridge_ip` call: I don't want to introduce too many changes to the `weave` script in this PR, so I'll get rid of it in a separate PR.